src/main/java/org/generationcp/middleware/dao/WorkbenchUserDAO.java

```diff
@@ -118,12 +121,14 @@ public boolean isUsernameExists(final String username) {
 	}
 	
 	@SuppressWarnings({"unchecked"})
-	public List<UserDto> getUsersByProjectUUId(final String projectUUID) {
+	public List<UserDto> getUsersByProjectUUId(final String projectUUID, final String cropName) {
```

Comment:
If we consolidate to one query with getting by project id, we don't need **cropName** parameter as well